haechan45th/src/services/userService.js

```diff
@@ -25,7 +35,23 @@ const signUp = async (name, email, password, profileImage) => {
     }
 };
 
+const signIn = async (email, password) => {
+    try {
+        const user = await userDao.getUserByEmail(email);
+        if (!user || !await bcrypt.compare(password, user.password)) {
+            throw new Error('KEY_ERROR');
+        }
```

Comment:
```suggestion
        if (!user || !await bcrypt.compare(password, user.password)) {
            throw new Error('INVALID_EMAIL_OR_PASSWORD');
        }
```
해당 케이스가 key error는 아니므로 보다 적절한 메세지를 작성하도록 수정되어야 합니다.